src/FormFields/SessionManagerField.php

```diff
@@ -127,14 +127,17 @@ protected function getLoginSessions(Member $member)
             if (!$loginSession->canView()) {
                 continue;
             }
+
+            $member = DataObject::get_by_id(Member::class, $loginSession->MemberID);
+
             $loginSessions[] = [
                 'ID' => $loginSession->ID,
                 'IPAddress' => $loginSession->IPAddress,
                 'UserAgent' => $loginSession->getFriendlyUserAgent(),
                 'IsCurrent' => $loginSession->isCurrent(),
                 'Persistent' => $loginSession->Persistent,
                 'Member' => [
-                    'Name' => $loginSession->Member()->Name
+                    'Name' => $member->Name
```

Comment:
```suggestion
                    'Name' => Member::get_by_id($loginSession->MemberID)->Name
```

Reply:
Is it safe to assume there's always a member?
```suggestion
                    'Name' => Member::get_by_id($loginSession->MemberID)->Name ?? ''
```

Reply:
It has been assuming that so far, so probably.

Reply:
worst case you get an empty object